### DISCUSSION TITLE
Why isn't a custom tag created?

### DISCUSSION THREAD

**Discussion by omqmo** [remarkjs/remark#866] [General] [answered]
Why isn't a custom tag created?
`ctrl + s to execute code`

https://codesandbox.io/s/practical-solomon-5ubdf?file=/src/index.js

**Comment by wooorm** [comment, marked as answer]
You seem to deal with markdown and HTML. In that case, this guide might help: https://unifiedjs.com/learn/recipe/remark-html/.

**Reply by omqmo** [reply]
> You seem to deal with markdown and HTML. In that case, this guide might help: https://unifiedjs.com/learn/recipe/remark-html/.

Thank you, it works. But there was another problem-nodes with the "html" type are not compiled. Why?

**Reply by wooorm** [reply]
See the third and fourth section there: "How to allow HTML embedded in markdown" and the next one.

**Reply by omqmo** [reply]
> See the third and fourth section there: "How to allow HTML embedded in markdown" and the next one.

It doesn't work. And I should clarify, I have html not in markdown, namely nodes. 
```
const node = {
    type: "html",
    value: "<div>...</div>",
    data: {
          hName: "html" // [*]
    }
}

**Reply by omqmo** [reply]
> See the third and fourth section there: "How to allow HTML embedded in markdown" and the next one.

Thank! it didn't work because of hName. :)
